### PR TITLE
Implement Failure Exporter

### LINKED ISSUE #24: Implement MTTR
Two use cases:

1. Automated: Pods go into crash loops and other failures observable from kubernetes
2. Human defined: Some sort of incident ticket getting open or closed. For now we could use GitHub issues to simulate this.

### LINKED ISSUE #35: Create an exporter that captures failure
Acceptance criteria:

- An exporter that captures failure event timestamps (start and end)
- Recommend to pull from a Ticketing System, like Jira
- An update to the dashboard that does the math between the Change Failure / Deployment Frequency

### PR DESCRIPTION
**Pull request by themoosman**
Add Failure exporter.

resolves #24 
resolves #35

### CODE REVIEW

**Comment by themoosman** [comment]
This also resolves issue #24

**Comment by themoosman** [comment]
resolves #35

**Comment by themoosman** [comment]
resolves #24, #35

File: exporters/failure/values.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Values file for exporter helm chart
+app_name: failure-exporter
+source_context_dir: exporters/
+app_file: failure/app.py
+source_ref: master
+source_url: https://github.com/redhat-cop/pelorus.git
+#source_url: https://github.com/themoosman/pelorus.git
```
Comment (etsauer) [inline exporters/failure/values.yaml:7]: don't forget to remove this.

**Pull request merged**